test/unit/lib/email-service-spec.js

```diff
@@ -66,6 +66,22 @@ describe('Email Service', () => {
       EmailService.prototype._includeDate.should.not.have.been.called;
     });
 
+    it('sets subject to email-service instance if passed object', () => {
+      const subject = {
+        customer: '',
+        caseworker: ''
+      };
+      emailService = new EmailService({data, subject});
+      emailService.subject.should.be.equal(subject);
```

Comment:
ok, but you'd need to use `deep` to test the values of each member in the object, right?